README.md

```diff
@@ -93,7 +93,7 @@ mvn clean
 Have a fantastic feature idea?  Spot a bug?  We would absolutely love for you to contribute to this project!  Please feel free to:
 
 * Fork this project
-* Create a feature branch from the <strong>dev</strong> branch
```

Comment:
no dev branch... 🤔

Reply:
You're right, feature branch should be from the current `master` branch. Can you fix that instead of deleting the line?

Reply:
You're right, thanks for the fix!